tensorflow/core/kernels/matmul_op.cc

```diff
@@ -580,13 +580,14 @@ struct MatMulFunctor<SYCLDevice, T> {
 
 #if defined(INTEL_MKL) && defined(ENABLE_MKL)
 
-// MKL does not support half, bfloat16 and int32 types for
+// MKL does not support half, bfloat16, int32 and int64 types for
```

Comment:
For completeness, could you please add that MKL's `MatMul` kernels for `float`, `double`, `complex64`, and `complex128` are registered in `mkl_matmul_op.cc` before this sentence?

Reply:
@penpornk Done.